CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+## General contribution guidelines
+
+* Have a look at open issues. Pick an unassigned issue. If doubtful, discuss on the issue itself.
+* You can also create a new issue for a feature or bug.
+* Make sure you are assigned to the issue before you start working on it.
+* Make sure you are not working on more than 2 issues at once.
+* While sending a PR make sure you follow one issue per PR rule.
+* Squash your commits in all PRs you send.
+* Make sure the Issue and PR template are correctly filled.
```

Comment:
Provide a template

Reply:
Don't think there is anything special to add on that.

Reply:
We can have some checks like [ ] which users can mark to verify some points like they are already not working on more than two issues and they checked the contribution guidelines etc.